scripts/y2k_birthday_update_query.json

```diff
@@ -0,0 +1,9 @@
+{
+    "query": {
+        "exists": { "field": "date_of_birth" }
```

Comment:
will the "exists" filter, filter out records will null value for "date_of_birth".

Reply:
Yep! And `[]` and `[null]` and... :)